crates/conformance-test-utils/src/utils.rs

```diff
@@ -0,0 +1,58 @@
+use starknet::core::types::FieldElement;
+use starknet::core::utils::get_contract_address;
+
+/// Computes the StarkNet address of EVM address given the Kakarot address and Account Proxy Class
+/// Hash. Uses the same logic as get_contract_address with EVM as salt.
+pub fn compute_starknet_address(
+    kakarot_address: FieldElement,
+    account_proxy_class_hash: FieldElement,
+    evm_address: FieldElement,
+) -> FieldElement {
+    let constructor_calldata: Vec<FieldElement> = Vec::new();
+    get_contract_address(evm_address, account_proxy_class_hash, &constructor_calldata, kakarot_address)
+}
+
+#[cfg(test)]
+mod tests {
+    use starknet::core::crypto::compute_hash_on_elements;
+    use starknet::core::types::FieldElement;
+    use starknet::core::utils::normalize_address;
+
+    use super::compute_starknet_address;
+
+    #[test]
+    fn test_compute_starknet_address() {
```

Comment:
tbh I don't really understand the value of this test. You're actually testing that starknet-rs didn't mess up with their `get_contract_address` iiuc, which we shouldn't do. You probably want to test instead that a calling a compute_starknet_address on deployed kakarot gives the same result as this offchain util.

Reply:
Yeah I wasn't sure how to test this properly as was not comfortable with having an on-chain dependency to a deployed Kakarot contract. But yeah I see what you mean, would it then be good to change this to a call on a deployed Kakarot on goerli?